cypress/support/e2e.js

```diff
@@ -48,6 +50,43 @@ Cypress.Commands.add("addAliasToGraphRequest", operationName => {
   });
 });
 
+Cypress.Commands.add("getGridCellInfo", (col, row) =>
+  /*
+    It seeks for react fiber node to obtain Glide grid instance.
+    Within its ref, we have getBounds API which returns an info about given cell.
+
+    Browsing over the three starts from canvas. In case of changing that selector,
+    this function must be adjusted as well.
+  */
+  cy
+    .get(SHARED_ELEMENTS.dataGridTable)
+    .should("be.visible")
+    .wait(3000)
```

Comment:
```suggestion
ensureCanvasStatic(PRODUCTS_LIST.dataGridTable).then
```

Reply:
everywhere grid testid stays the same, so it's rather shared element than contexted just for the products